moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -58,24 +58,29 @@ namespace collision_detection
 {
 MOVEIT_STRUCT_FORWARD(CollisionGeometryData);
 
+/** \brief Wrapper around world, link and attached objects' geometry data. */
 struct CollisionGeometryData
 {
+  /** \brief Constructor for a robot link collision geometry object. */
   CollisionGeometryData(const robot_model::LinkModel* link, int index) : type(BodyTypes::ROBOT_LINK), shape_index(index)
   {
     ptr.link = link;
   }
 
+  /** \brief Constructor for a new collision geometry object which is attached to the robot. */
   CollisionGeometryData(const robot_state::AttachedBody* ab, int index)
     : type(BodyTypes::ROBOT_ATTACHED), shape_index(index)
   {
     ptr.ab = ab;
   }
 
+  /** \brief Constructor for a new world collision geometry. */
   CollisionGeometryData(const World::Object* obj, int index) : type(BodyTypes::WORLD_OBJECT), shape_index(index)
   {
     ptr.obj = obj;
   }
 
+  /** \brief Returns the name which is saved in the pointed to member \e ptr. */
```

Comment:
```suggestion
  /** \brief Returns the name which is saved in the member pointed to in \e ptr. */
```

or "pointed-to member"

Reply:
done